pkg/kube/wait.go

```diff
@@ -63,6 +63,7 @@ func (w *waiter) waitForResources(created ResourceList) error {
 			ready, err := w.c.IsReady(ctx, v)
 
 			if waitRetries > 0 && w.isRetryableError(err, v) {
+				waitRetries--
```

Comment:
Why is this being removed?

Reply:
@gjenkins8 the logic was originally added incorrectly, `waitRetries` don't decrease on each try, it's only increase so the `waitRetries > 0` condition is always true